src/members/codeshow.md

```diff
@@ -14,34 +14,27 @@ tags:
   - Linux
   - Flask
 schedule:
-  - Tuesday at 9:30pm BRT
-  - Thursdays at 9:30pm BRT
-  - Saturdays at a random time
+  - Tuesday at a random time.
+  - Thursdays at 21:30 BRT
+  - Saturdays at 14:00 BRT
 ---
 
-# About
+![Bruno Rocha](https://raw.githubusercontent.com/rochacbruno/rochacbruno/master/img/redhat_avatar2.png)
```

Comment:
So would you want the image to be here. When I previewed it, it has this image basically two times and I'm unsure if we need it both as the avatar and in the body copy

![Screenshot_2020-07-24 codeshow The Live Coders](https://user-images.githubusercontent.com/3685876/88418519-a9ce5a80-cdb1-11ea-8dac-c90d6a508438.png)